test/unit/a-url-error/input.js

```diff
@@ -0,0 +1 @@
+const URLParser = typeof window === 'undefined' ? URL : 'b'
```

Comment:
Is this a regression from PR https://github.com/vercel/nft/pull/429 ?

It added url to global, although this test doesn't have `require('url')` so may not be related.

Or maybe it was from the URL global here?

https://github.com/vercel/nft/blob/b3d2da0ef01c25b5d99720a70fefbbd09c9b844c/src/analyze.ts#L187

Reply:
More info on this:
- The URL global you linked does seem to be the culprit!
- Removing URL from globals makes this case work, but breaks other cases
- The core problem exists well before `0.27.4`, but `0.27.4` seems to trace more files which causes our test to hit this URL core problem now

Debugging this is tricky since the stack trace is very unhelpful. For example, replacing `URL: URL` with `URL: () => console.trace('Called URL')` gives the very unhelpful stack trace:
```
Trace: Called URL
    at Object.URL [as then] (/Users/onsclom/projects/nft/src/analyze.ts:188:13)
```

I can't figure out where URL is getting called from. Seems like from the `then` of some promise, but which one?

Reply:
@styfle Do you know how URL is getting invoked here? Happy to make a PR fixing this, but stuck on finding where exactly URL is getting invoked.

Reply:
Its probably from `computePureStaticValue()` which is basically a static analysis form of `eval()` so it sets globals vars and local vars to try to compute a value.

https://github.com/vercel/nft/blob/b3d2da0ef01c25b5d99720a70fefbbd09c9b844c/src/analyze.ts#L459